lib/smack/SmackRep.cpp

```diff
@@ -109,6 +196,22 @@ string SmackRep::memType(unsigned r) {
   }
 }
 
+//
+// Memory type for bit-vectors
+//
+string SmackRep::memType(unsigned r, unsigned type) {
```

Comment:
I've merged memType(unsigned) to memType(unsigned, unsigned) so the latter one is the only memType method. Am I correct in doing so? SmackRep::mem method sets could also be reduced in this way while I wonder if merging on-bit-vector methods to bit-vector ones will cause confusion when you are using these methods.